lesson_03/homework_03.py

```diff
@@ -25,7 +37,19 @@
 250 449 товарів. На другому та третьому – 222 950 товарів.
 Знайдіть кількість товарів, що розміщені на кожному складі.
 """
-
+# my answer:
+total_amount = 375291
+stores_1_2 = 250449
+stores_2_3 = 222950
+store_1 = total_amount - stores_2_3
+store_2 = stores_1_2 - store_1
+store_3 = stores_2_3 - store_2
+print(f'''If total_amount = 375291
```

Comment:
```suggestion
print(f'''If total_amount = {total_amount}
```
і так далі, тоді завжди будуть вірні значення